.github/actions/ci/action.yml

```diff
@@ -7,6 +7,14 @@ inputs:
   test_project_file:
     description: 'Path to the test project file.  If provided, run unit tests, otherwise skip them.'
     required: false
+  target_test_framework:
+    description: 'The target test framework to use.'
+    required: false
+    default: 'net6.0'
+  test_verbosity:
```

Comment:
Without this, it just appears like the tests are hanging in CI. I can see the value of not having this when running locally, but it's nice to have the record when waiting remotely.